package.json

```diff
@@ -32,15 +32,16 @@
     "sinon": "^4.4.2"
   },
   "dependencies": {
+    "@webcomponents/url": "git+https://github.com/webcomponents/URL.git",
     "async": "~0.2.6",
     "babel": "^6.3.26",
     "babel-cli": "^6.22.2",
     "babel-core": "^6.22.1",
     "babel-eslint": "~8.1.2",
     "babel-loader": "^6.2.9",
     "babel-plugin-array-includes": "^2.0.3",
-    "babel-plugin-transform-class-properties": "^6.24.1",
     "babel-plugin-transform-async-to-generator": "^6.24.1",
+    "babel-plugin-transform-class-properties": "^6.24.1",
```

Comment:
One minor note: looks `"babel-plugin-transform-class-properties": "^6.24.1` was in previous version already. Does order matter?

Reply:
No the order is irrelevant. 
That was just my auto-formatter keeping the list in alphabetical order.